schemas_src/abstract/MPModel.json

```diff
@@ -22,23 +16,11 @@
         "prototype": {
           "$ref": "strings.json#/definitions/_id"
         },
-        "locked": {
-          "type": "boolean"
-        },
-        "bundled": {
-          "type": "boolean"
-        },
         "keywordIDs": {
           "type": "array",
           "items": {
             "$ref": "strings.json#/definitions/keywordID"
           }
-        },
-        "contributions": {
```

Comment:
As far as I know, this property is in use. 
I can recall using it while constructing comments

Reply:
It looks like we are using it only for comments, so I added it back under the CommentAnnotation type.